### ISSUE TITLE
Security Advisory: Vulnerability in `phin` Dependency

### ISSUE THREAD

**Issue by babblebey** [ljharb/ls-engines#30]
I received a security advisory indicating that there is a moderate severity vulnerability in the `phin` dependency used by the `get-json` dependency in the `ls-engines` package. The `phin` package may include sensitive headers in subsequent requests after a redirect. There is currently no fix available for this vulnerability.

**Audit Report:**
```
# npm audit report

phin  <3.7.1
Severity: moderate
phin may include sensitive headers in subsequent requests after redirect - https://github.com/advisories/GHSA-x565-32qp-m3vf
No fix available
node_modules/phin
  get-json  1.0.0 - 1.0.1
  Depends on vulnerable versions of phin
  node_modules/get-json
    ls-engines  *
    Depends on vulnerable versions of get-json
    node_modules/ls-engines

3 moderate severity vulnerabilities

Some issues need review, and may require choosing
a different dependency.
```

**Dependency Tree:**
```
semantic-release@0.0.0-development
└─┬ ls-engines@0.9.1
  └─┬ get-json@1.0.1
    └── phin@2.9.3
```

**Steps to Reproduce:**
1. Run `npm audit` to see the vulnerability report.
2. Run `npm ls phin` to view the dependency tree involving the `phin` package.

**Context:**
We are using `ls-engines` as a development dependency in our `semantic-release` project. While this does not directly affect our users, it is important to address the vulnerability for the security and integrity of our development environment.

**References:**
- https://github.com/semantic-release/semantic-release/security/dependabot/33

**Comment by ljharb** [comment]
It's not actually a vulnerability, because get-json is only used with a hardcoded URL here, and there's no headers to be concerned with.

That said, I'm in the process of reviving the `get-json` package, and I intend to publish a version that doesn't depend on `phin`, so the noisy audit warning will go away eventually.

**Comment by babblebey** [comment]
Sounds good 😁

**Comment by ljharb** [comment]
Fixed in a7f14a3, which should also Just Work if you reinstall (v1.1.0 of get-json no longer uses phin).
